Shared/OptionsFile.cs

```diff
@@ -75,6 +75,7 @@ public bool RequireBool(string key)
 
             try
             {
+
```

Comment:
The man changes in this file seem to be newlines. Any reason for those? Is that according to some c# formatting guide? If yes, I'm happy to include them.

Reply:
I think that happened when I tried to resolve a conflict. When I forked your branch this wasn't fixed, so in my original pr I included a insecure quick fix. now that you've taken care of that I didn't have to have touched that file.